docs/rest.md

```diff
@@ -0,0 +1,622 @@
+# Snappy Ubuntu Core REST API
+
+Version: 1.0.1 DRAFT (look for "not implemented" to find bits still in flux or
+not implemented)
+
+## Versioning
+
+As the API evolves, some changes are deemed backwards-compatible (such
+as adding methods or verbs, or adding members to the returned JSON
+objects) and don’t warrant an endpoint change; some changes won’t be,
+and will be exposed under a new endpoint.
+
+## Connecting
+
+While it is expected to allow clients to connect using HTTPS over a
+TCP socket, as described above, at this point only a unix socket is
+supported. The socket is `/run/snapd`.
+
+## Authentication
+
+Authentication over the unix socket is delegated to UNIX ACLs. At this point
+only root can connect to it; later, regular user access is not implemented
+yet, but should be able to once SO_PEERCRED is supported to determine
+privilege levels.
```

Comment:
Do you have a bug number to track the work of SO_PEERCRED? That would be a useful link.